specifications/08-variants.json

```diff
@@ -339,6 +360,21 @@
                 },
                 "enabled": true
             }
+        },
+        {
+            "description": "Feature.Variants.E should be disabled",
+            "context": {
+                "userId": "0"
+            },
+            "toggleName": "Feature.Variants.E",
+            "expectedResult": {
+                "name": "variant1",
+                "payload": {
+                    "type": "string",
+                    "value": "val1"
+                },
+                "enabled": false
+            }
```

Comment:
The disabled variant should actually take the shape of:

```json
{
    "name": "disabled",
    "enabled": false
}
```

(no payload)

Reply:
That's been corrected now, thanks.